code/modules/cm_aliens/XenoStructures.dm

```diff
@@ -698,6 +698,8 @@ TUNNEL
 /obj/structure/tunnel/Destroy()
 	GLOB.xeno_tunnels -= src
 	if(other)
+		other.health = 0 //No sense in keeping the defunct tunnel around.
+		other.healthcheck()
```

Comment:
This will delete the object referenced by `other`.
Then `other.other = null` will runtime.
This is the third locally-untested PR causing errors in less than a week.

Reply:
Damn, I feel stupid now.

Reply:
yeah this is problematic